includes/ExternalUser.php

```diff
@@ -129,20 +129,23 @@ public static function newFromUser( $user ) {
 	/**
 	 * add new User object to database
 	 * 
-	 * @param $user User
-	 * @return mixed User or false
+	 * @param $User User an instance of User to add
```

Comment:
Not sure if its worth it for this (dying) class, but the variable name and type are out of order (http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam - see last part of this section for PHP info)